rita_client/src/exit_manager/exit_loop.rs

```diff
@@ -334,12 +334,13 @@ fn update_exit_server_to_althea_addresses() {
                         deep_space::Address::from_slice(e.eth_address.as_bytes(), "althea")
                             .expect("This shouldnt be failing"),
                     );
+                    info!("Updated us west alteha address to {:?}", e.althea_address)
                 }
             }
             _ => {}
         }
     }
-    // Save into settings
+    // save new rita client
```

Comment:
just to clarify this does not actually save the settings to disk but only updates the settings in the memory var

Reply:
updated wording here